layers/+completion/helm/packages.el

```diff
@@ -256,6 +257,13 @@
       (add-hook 'helm-mode-hook 'helm-descbinds-mode)
       (spacemacs/set-leader-keys "?" 'helm-descbinds))))
 
+(defun helm/init-helm-exwm ()
```

Comment:
Can we add this conditionally so that it is only added if the exwm layer is added?

Reply:
Will do.